log/log.go

```diff
@@ -30,6 +30,10 @@ func Err(msg string) {
 	fmt.Fprintln(os.Stderr, msg)
 }
 
+func Errf(msg string, args ...interface{}) {
+	fmt.Fprintf(os.Stderr, msg, args...)
```

Comment:
Nitpick: this is changed by two different commits :mag:

Reply:
Shit, squash blunder